tests/robot-cases/Group1-Nightly/OIDC.robot

```diff
@@ -0,0 +1,66 @@
+// Copyright Project Harbor Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+*** Settings ***
+Documentation  Harbor BATs
+Resource  ../../resources/Util.robot
+Default Tags  Nightly
+
+*** Variables ***
+${HARBOR_URL}  https://${OIDC_HOSTNAME}
+${SSH_USER}  root
+${HARBOR_ADMIN}  admin
+
+*** Test Cases ***
+Test Case - Get Harbor Version
+#Just get harbor version and log it
+    Get Harbor Version
+
+Test Case - OIDC User Sign In
+    Sign In Harbor With Ldap User    ${HARBOR_URL}
+    Sleep  2
+    Sign In Harbor With Ldap User    ${HARBOR_URL}    test2
+    Sleep  2
+    Sign In Harbor With Ldap User    ${HARBOR_URL}    test3
+    Sleep  2
+    Sign In Harbor With Ldap User    ${HARBOR_URL}    test4
+    Sleep  2
+    Sign In Harbor With Ldap User    ${HARBOR_URL}    test5
+    Sleep  2
+    Sign In Harbor With Ldap User    ${HARBOR_URL}    test6
+    Sleep  2
+    Sign In Harbor With Ldap User    ${HARBOR_URL}    test7
+    Sleep  2
+    Sign In Harbor With Ldap User    ${HARBOR_URL}    test8
+    Sleep  2
+    Sign In Harbor With Ldap User    ${HARBOR_URL}    test9
+    Sleep  2
+    Close Browser
+
+Test Case - Create An New Project
+    Sign In Harbor With Ldap User    ${HARBOR_URL}
+    ${d}=    Get Current Date    result_format=%M%S
+    Create An New Project  test${d}
+    Close Browser
+
+Test Case - Delete A Project
+    Init Chrome Driver
+    Sign In Harbor With Ldap User    ${HARBOR_URL}
+    ${json}=  Run Curl And Return Json  curl -s -k -X GET --header 'Accept: application/json' -u '${HARBOR_ADMIN}:${HARBOR_PASSWORD}' 'https://10.192.123.153/api/users/search?username=${OIDC_USERNAME}'
```

Comment:
why hard code the IP address? it's should be an internal one, don't expose to the public world.